app/routes/joblistings/components/JoblistingRightNav.tsx

```diff
@@ -150,64 +150,44 @@ const JoblistingsRightNav = (props: Props) => {
             <Button>Ny jobbannonse</Button>
           </Link>
         )}
+
         <h3 className={styles.rightHeader}>Sorter etter:</h3>
-        <label
-          htmlFor="deadline"
-          style={{
-            cursor: 'pointer',
+        <RadioButton
+          name="sort"
+          id="deadline"
+          label="Frist"
+          checked={order.deadline}
+          onChange={() => {
+            props.history.push({
```

Comment:
Would be nice if you could replace these with `replace` as well :smile:

Reply:
Out of scope here. Write it up as a todo and we can go over all of these